apps/ui/src/composables/useAiSummary.ts

```diff
@@ -0,0 +1,34 @@
+const SIDEKICK_URL = 'https://sh5.co';
+
+export function useAiSummary(proposalId: string) {
+  const aiSummary = ref<string>('');
+  const state = ref({
+    loading: false,
+    error: null,
+    open: false
+  });
+
+  async function fetchAiSummary() {
+    if (!aiSummary.value) {
```

Comment:
```suggestion
    if (aiSummary.value) return
```

And we avoid indentations in this whole block.